jquery.fancytree/index.d.ts

```diff
@@ -199,7 +202,7 @@ declare namespace Fancytree {
         /** Display name (may contain HTML) */
         title: string;
         /** Contains all extra data that was passed on node creation */
-        data: Object;
+        data: any;
```

Comment:
Why is ```any``` necessary here? Could you add some usage examples in the test file to demonstrate?